docs/source/trainer/numerics.md

```diff
@@ -0,0 +1,73 @@
+# 🔢 Numerics
+
+Composer supports several single and half-precision number formats including IEEE single-precision `fp32`, IEEE half-precision `fp16`, Google’s truncated `bf16` format and support for mixed precision training `amp`.  The following formats are supported per accelerator:
+
+| Format | CPU | GPU - A100 | GPU - V100 | GPU - T4 |
+| --- | --- | --- | --- | --- |
```

Comment:
Nit: can you format the table so the columns align?

Reply:
I cleaned it up, won't be perfect bc the inline emoji's aren't same fixed fidth.